egs/tedlium/s5_r3/local/chain/run_tdnn_1c.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+
+# 1g is as 1f but moving to a factorized TDNN (TDNN-F) model, re-tuning it, and
```

Comment:
Please make sure this header is fully up to date with the current results.  E.g. you can add a comparison with whatever script was the default before, like 1b.